bindata/assets/config/defaultconfig.yaml

```diff
@@ -14,8 +14,12 @@ admission:
         kind: PodSecurityConfiguration
         apiVersion: pod-security.admission.config.k8s.io/v1beta1
         defaults:
-          enforce: "restricted"
+          enforce: "privileged"
```

Comment:
in master we want the enforcement to be the default

Reply:
> in master we want the enforcement to be the default

The point of this change is to be able to configure it, so this is overwritten either way. Otherwise your backports will fail.